### PR TITLE
SEO-187489-ej2-Asp.net-core-Introduction-Page-with-redirect-link

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | Coverage Issues|
|Content Task Link/Mail Screenshot | NA|
|Hotfix |https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/3276|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187647|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B2A368E74-D539-456F-9819-C9B68637E490%7D&file=Faith.xlsx&action=default&mobileredirect=true|

Title | Description
-- | --
|Task Category | Coverage Issues|
|Content Task Link/Mail Scr
Changes made | Reason for change
-- | --
| Page with redirect| To increase page performance |

### CODE REVIEW

**Comment by Faith-Atieno** [comment]
rebuild

File: ej2-asp-core-mvc/EJ2_ASP.NETCORE/introduction.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Introduction of Syncfusion ##Platform_Name## controls
+title: Introduction of Syncfusion ##Platform_Name## controls | Syncfusion
```
Comment (Yvone-Atieno) [inline ej2-asp-core-mvc/EJ2_ASP.NETCORE/introduction.md:3]: Introduction of Syncfusion ##Platform_Name## Controls | Syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-asp-core-mvc/EJ2_ASP.NETCORE/introduction.md (hunk 1 above)
Comment (Faith-Atieno) [reply ej2-asp-core-mvc/EJ2_ASP.NETCORE/introduction.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**